cmd/ocagent/config.go

```diff
@@ -17,10 +17,9 @@ package main
 import (
 	"log"
 
-	yaml "gopkg.in/yaml.v2"
-
 	"github.com/census-instrumentation/opencensus-service/exporter"
 	"github.com/census-instrumentation/opencensus-service/exporter/exporterparser"
+	yaml "gopkg.in/yaml.v2"
```

Comment:
Please revert this line, keep this import separated from the "github.com/census-instrument*" imports.